src/pages/projects-list/ProjectsList.tsx

```diff
@@ -158,31 +156,10 @@ const ProjectsList: React.FC = () => {
                 list={list}
                 disableMultiplePageSelection={true}
                 buttonLabel={i18n.t("Create Project")}
-                customFiltersComponent={() => (
-                    <CustomFilters
-                        createdByCurrentUser={byUser}
-                        onChange={() => setByUser(!byUser)}
-                    />
-                )}
-                customFilters={{ createdByCurrentUser: byUser }}
```

Comment:
That's correct, we don't need the `byUser` flag anymoire, but we still need to filter, because we want to display only the user's projects. We could simply write `customFilters={{ createdByCurrentUser: true }}`, but we are on a refactor on this component and this prop is probably going away. So let's add the code to the listing call. A proposal:

```
 const list = (_d2: unknown, filters: FiltersForList, pagination: Pagination) => {
        const fullFilters = { ...filters, createdByCurrentUser: true };
        return Project.getList(api, fullFilters, pagination);
    };
```

Reply:
I am fine with the implementation but for now (I guess this will change in the future) we want to list all the projects, not only the user's ones. So same thing but with a 'false' :)

Reply:
Should we then update #22? 

> We should display only the user's project

Reply:
Yep, I wrote it wrong!

Reply:
So then, fine by me!

Reply:
I think what @tokland is proposing is to add something like his proposed bit of code here https://github.com/EyeSeeTea/project-monitoring-app/blob/22-remove-my-projects-filter/src/pages/projects-list/ProjectsList.tsx#L137